src/app/components/button/button.module.css

```diff
@@ -0,0 +1,21 @@
+.button--primary {
+  background-color: #3c3db9;
+  width: 200px;
```

Comment:
If a max-width isn't needed, consider letting the button adjust its size based on the text from the component prop. This can be done by setting appropriate padding, rather than an explicit width here.

Reply:
this is my usual approach but with tailwind, i was following the guidelines in the original issue. I agree though, i'll make this change!